### PR TITLE
new in-house 2pt DOSTA cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/DOSTAD/CGINS-DOSTAD-00514__20240131.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+514,CC_conc_coef,"[-6.456627E-01, 1.050923E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_514_Calibration_Certificate_2024-01-31.pdf"
+514,CC_csv,"[2.53888E-03, 1.08423E-04, 2.15603E-06, 1.68142E+02, -2.51008E-01, -4.29735E+01, 3.39845E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00514__20240131.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00516__20240117.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+516,CC_conc_coef,"[-2.515070E+00, 1.007233E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_516_Calibration_Certificate_2024-01-17.pdf"
+516,CC_csv,"[2.68171E-03, 1.17343E-04, 2.17994E-06, 1.74229E+02, -2.24825E-01, -4.20225E+01, 3.42855E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00516__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00517__20240117.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+517,CC_conc_coef,"[-1.849033E+00, 1.016277E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_517_Calibration_Certificate_2024-01-17.pdf"
+517,CC_csv,"[2.61075E-03, 1.17782E-04, 2.07297E-06, 1.70117E+02, -2.70562E-01, -4.49428E+01, 3.39244E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00517__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00518__20240118.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+518,CC_conc_coef,"[-2.339728E+00, 1.022990E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_518_Calibration_Certificate_2024-01-18.pdf"
+518,CC_csv,"[2.71076E-03, 1.17758E-04, 2.23528E-06, 1.93853E+02, -2.67531E-01, -4.84075E+01, 3.81209E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00518__20240118.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00663__20240131.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+663,CC_conc_coef,"[-1.291427E+00, 1.066409E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_663_Calibration_Certificate_2024-01-31.pdf"
+663,CC_csv,"[2.81476E-03, 1.18957E-04, 2.37181E-06, 2.30050E+02, -3.88265E-01, -5.77627E+01, 4.55125E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00663__20240131.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00502__20240117.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+502,CC_conc_coef,"[-2.306866E-01, 1.011142E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_502_Calibration_Certificate_2024-01-17.pdf"
+502,CC_csv,"[2.65279E-03, 1.13001E-04, 2.24904E-06, 9.12452E+01, -1.45210E-01, -2.24209E+01, 1.81645E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00502__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00227__20240201.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+227,CC_conc_coef,"[-3.694623E+00, 1.170710E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_227_Calibration_Certificate_2024-02-01.pdf"
+227,CC_csv,"[3.11358E-03, 1.30733E-04, 2.50351E-06, 2.32461E+02, -3.46572E-01, -5.80057E+01, 4.53733E+00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00227__20240201.csv:3]: looks good

**Review by cooleyky — COMMENTED** [review]
One possible typo in S/N 126, otherwise looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00126__20240208.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+126,CC_conc_coef,"[-3.410997E+00, 1.068534E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_126_Calibration_Certificate_2024-02-08.pdf"
+126,CC_csv,"[2.68474E-03, 1.17084E-04, 2.23543E-06, 1.98627E+02, -2.16587E-01, -4.89511E+01, 3.94502E+00]",
```
[the remark this answers is not in the document]
Comment (cooleyky) [reply calibration/DOSTAD/CGINS-DOSTAD-00126__20240208.csv:3]: all coefficients match a previous vendor cal file. looks good

**Review by cooleyky — APPROVED** [review]
everything looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00463__20240209.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+463,CC_conc_coef,"[2.063940E-01, 9.933791E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_463_Calibration_Certificate_2024-02-09.pdf"
+463,CC_csv,"[2.58528E-03, 1.10840E-04, 2.16400E-06, 1.62108E+02, -2.46740E-01, -3.98702E+01, 3.22007E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00463__20240209.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00465__20240130.csv [hunk 10]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+465,CC_conc_coef,"[-4.298867E+00, 1.035267E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_465_Calibration_Certificate_2024-01-30.pdf"
+465,CC_csv,"[2.59425E-03, 1.09111E-04, 2.24929E-06, 2.06549E+02, -2.62493E-01, -5.08780E+01, 4.10097E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00465__20240130.csv:3]: looks goood

File: calibration/DOSTAD/CGINS-DOSTAD-00466__20240201.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+466,CC_conc_coef,"[-3.043675E+00, 1.085236E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_466_Calibration_Certificate_2024-02-01.pdf"
+466,CC_csv,"[2.85179E-03, 1.21990E-04, 2.37240E-06, 2.32010E+02, -3.09932E-01, -5.43607E+01, 4.56618E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00466__20240201.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00467__20240129.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+467,CC_conc_coef,"[-1.767053E+00, 1.030906E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_467_Calibration_Certificate_2024-01-29.pdf"
+467,CC_csv,"[2.60720E-03, 1.08657E-04, 2.23171E-06, 1.97483E+02, -2.53546E-01, -4.44708E+01, 3.90092E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00467__20240129.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00497__20240118.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+497,CC_conc_coef,"[-3.582432E+00, 1.052999E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_497_Calibration_Certificate_2024-01-18.pdf"
+497,CC_csv,"[2.56178E-03, 1.07341E-04, 2.29944E-06, 1.99134E+02, -2.46262E-01, -5.11101E+01, 3.97527E+00]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/DOSTAD/CGINS-DOSTAD-00497__20240118.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00498__20240207.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+498,CC_conc_coef,"[-2.352778E+00, 1.016393E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_498_Calibration_Certificate_2024-02-07.pdf"
+498,CC_csv,"[2.56649E-03, 1.10088E-04, 2.07319E-06, 2.06953E+02, -2.50288E-01, -4.73333E+01, 4.07663E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00498__20240207.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00502__20240117.csv (hunk 6 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00502__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00514__20240131.csv (hunk 1 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00514__20240131.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00516__20240117.csv (hunk 2 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00516__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00517__20240117.csv (hunk 3 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00517__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00518__20240118.csv (hunk 4 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00518__20240118.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00663__20240131.csv (hunk 5 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00663__20240131.csv:3]: looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**